README.md

```diff
@@ -68,17 +55,19 @@ new ecrdeploy.ECRDeployment(this, 'DeployDockerImage3', {
 ## Sample: [test/example.ecr-deployment.ts](./test/example.ecr-deployment.ts)
 
 ```shell
+# Generate crane lambda layer firstly.
+./layer/build.sh
```

Comment:
This section needs some clarification that these instructions only apply if you're developing on the repository, not if you're a user of the construct.

`README.md` is typically consumer-targeted, and the section title says "Sample", so it should (mostly) only contain instructions for consumers.

These instructions would make more sense in `CONTRIBUTING.md` in a section title "How to run tests".

If you want to keep them here, clearly label the instructions as intended for developers, not for consumers.